catroid/build.gradle

```diff
@@ -26,7 +26,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.3'
```

Comment:
Put please don't commit these change. There can be problems with jenkins, and so on. The gradle version will only be updated by the coordinators.